src/payment/strategies/adyenv2/adyenv2-script-loader.ts

```diff
@@ -2,29 +2,31 @@ import { ScriptLoader, StylesheetLoader } from '@bigcommerce/script-loader';
 
 import { PaymentMethodClientUnavailableError } from '../../errors';
 
-import { AdyenCheckout, AdyenConfiguration, AdyenHostWindow } from './adyenv2';
+import { AdyenClient, AdyenConfiguration, AdyenHostWindow } from './adyenv2';
 
 export default class AdyenV2ScriptLoader {
     constructor(
         private _scriptLoader: ScriptLoader,
         private _stylesheetLoader: StylesheetLoader,
         private _window: AdyenHostWindow = window
-    ) {}
+    ) { }
 
-    load(configuration: AdyenConfiguration): Promise<AdyenCheckout> {
-        return Promise.all([
-            this._stylesheetLoader.loadStylesheet(`https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.6.0/adyen.css`),
-            this._scriptLoader.loadScript(`https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.6.0/adyen.js`),
-        ])
-        .then(() => {
-            if (!this._window.AdyenCheckout) {
-                throw new PaymentMethodClientUnavailableError();
-            }
+    load(configuration: AdyenConfiguration): Promise<AdyenClient> {
+        const promises = [];
+
+        promises.push(this._stylesheetLoader.loadStylesheet(`https://checkoutshopper-${configuration.environment}.adyen.com/checkoutshopper/sdk/3.6.0/adyen.css`));
```

Comment:
using `push()` it's better suited for when you have conditional promises that are not always meant to be executed, e.g.
```js
if (X) {
  promises.push(/** .... */)
}
```

In this case, might be better to keep what it was there before
```js
await Promise.all([
  // ...,
  // ...
]);

if (!this._window.AdyenCheckout) {
  throw new PaymentMethodClientUnavailableError();
}

return new this._window.AdyenCheckout(configuration);
```